### DISCUSSION TITLE
@sveltejs/enhanced-img and GIF support

### DISCUSSION THREAD

**Discussion by alexbjorlig** [sveltejs/kit#11039] [Q&A]
I'm implementing `@sveltejs/enhanced-img`, but what is the expected behavior when loading a GIF?

Does it seem like the GIF is frozen when I use the `enhanced-image`?

Maybe we could do something like described [here](https://uploadcare.com/docs/transformations/gif-to-video/)?

**Comment by benmccann** [comment]
See https://github.com/JonasKruckenberg/imagetools/issues/317
